patches/net/minecraft/world/entity/LivingEntity.java.patch

```diff
@@ -9,6 +9,14 @@
      private static final Logger LOGGER = LogUtils.getLogger();
      private static final String TAG_ACTIVE_EFFECTS = "active_effects";
      private static final UUID SPEED_MODIFIER_SOUL_SPEED_UUID = UUID.fromString("87f46a96-686f-4796-b035-22e16ee9e038");
+@@ -242,6 +_,7 @@
+     protected Brain<?> brain;
+     private boolean skipDropExperience;
+     protected float appliedScale = 1.0F;
++    protected net.neoforged.neoforge.common.damagesource.DamageContainer.InternalDamageContainer damageContainer;
```

Comment:
Missing `@Nullable` annotation.  This should also include a javadoc comment about when this field is instantiated and when it returns to being null again.